inscriptions/ISic000787.xml

```diff
@@ -199,11 +199,11 @@
         <body>
             <div type="edition" xml:space="preserve" xml:lang="la" resp="#DEE">
                 <ab>
-                    <lb n="1"/><expan><abbr>D</abbr><ex>is</ex></expan> ❦ <expan><abbr>M</abbr><ex>anibus</ex></expan> ❦
-                    <lb n="2"/><expan><abbr>Ti</abbr><ex>berius</ex></expan> ❦ <expan><abbr>Cl</abbr><ex>audius</ex></expan>Claudianus
-                    <lb n="3"/><expan><abbr>vix</abbr><ex>it</ex></expan> ❦ <expan><abbr>ann</abbr><ex>is</ex></expan> ❦ XIIII
-                    <lb n="4"/><expan><abbr>Cl</abbr><ex>audius</ex></expan> ❦ Theseus ❦ pater
-                    <lb n="5"/>filio ❦ dulcissimo
+                    <lb n="1"/><expan><abbr>D</abbr><ex>is</ex></expan> <g ref="#ivy-leaf"/> <expan><abbr>M</abbr><ex>anibus</ex></expan> <g ref="#ivy-leaf"/>
```

Comment:
half right! We include the unicode symbol wherever it exists, so this should be:
`<g ref="#ivy-leaf">❦</g>`
We encode glyphs with an empty element only when the unicode symbol doesn't exist. Example of the appropraite unicode symbol will normally be included in the chardecl.xml file in the alist folder.